### ISSUE TITLE
Clear Date of Symptom onset when no symptoms are selected

### ISSUE THREAD

**Issue by raghupola-vg** [hzi-braunschweig/SORMAS-Project#5451]
### Feature Description

Clearing the existing symptoms, doesn't clear the Symptom onset date.
 
### Problem Description

When the Symptoms are cleared the Symptom onset date is not cleared and so the follow up date is also not changed. 

### Proposed Change

Clear the symptoms onset date when no symptoms are selected.

### Additional Information

![Disabled Date of Symptom onset date](https://user-images.githubusercontent.com/62763628/118464730-73c3e380-b701-11eb-9de8-c15ef78a4369.PNG)

**Comment by MateStrysewske** [comment]
@kwa20 @bernardsilenou Is this fine for you? We currently prevent users from entering an onset date if no symptom has been set to "Yes", so this would be consequential, but deleting an onset date that has already been entered might also not be desirable because it could lead to unintended data loss.

**Comment by kwa20** [comment]
@MateStrysewske I agree that simply deleting is not ideal, I'd suggest to maybe show a prompt because deleting the onset date also has influence on e.g. the follow-up period.
